roles/monitored/tasks/nrpe-install.yml

```diff
@@ -49,7 +49,7 @@
     owner: root
     group: root
     mode: 0755
-  with_filetree: monitoring/checks/
+  with_filetree: monitoring/checks/ # noqa 104 ansible-lint#582
```

Comment:
That PR has been merged..

Reply:
I don't think it's been included in a release - it looks like the last release was on the same day as the PR, but it didn't get in?